lib/riemann.js

```diff
@@ -113,11 +129,17 @@ Riemann.prototype.handlePacketEvent = function (eventString) {
 Riemann.prototype.send = function (eventData) {
   var eventToSend = this.client.Event(eventData);
 
-  // Send using proper transport
-  if (this.config.transport && this.config.transport === "tcp") {
-    this.client.send(eventToSend, this.client.tcp);
-  } else {
-    this.client.send(eventToSend);
+  try {
+    // Send using proper transport
+    if (this.config.transport && this.config.transport === "tcp") {
+      this.client.send(eventToSend, this.client.tcp);
+    } else {
+      this.client.send(eventToSend);
+    }
+  } catch (err) {
```

Comment:
I think I added that because there were cases when send() threw an exception, and other cases when client emited an error. I wanted to just log the error and drop the event (it would be a good idea to implement some retry queue, but this is outside of the scope of this quick fix)